CONTRIBUTING.md

```diff
@@ -73,3 +73,39 @@ After a pull request is approved, the branch should be deleted once merged.
    6. Verify tests have passed
 
 CONGRATULATIONS! You have successfully contributed to the Dallas Software Developers website! Now just wait and see if your pull request is approved.
+
+## When your Pull Request is merged
+
+That's Great, your work is well appreciated! If you would like to keep contributing, you should update your fork and update your local repository.
+
+## When your branch goes out of date
+
+This often happens and is mostly encountered after a pull request opened by another individual is merged to the `main` branch. Here are steps to follow if this happens:
+
+> You may choose to merge them the `main` branch on GitHub to your working branch which may be faster, but these steps are encouraged for a more linear-looking git history
+
+### Update your Fork
+
+1. Syncronize your Fork by going to your forked repository and ensure that you are on the `main` branch. You will see a message like { message }
+2. Click { buttonName } and click **Update Branch** on the prompt that appears.
+
```

Comment:
To add images and remove these button placeholders, I will need this branch to go out of date in order to take screenshots of my steps. This will be done when a pull request from another individual is merged to `main`.

Reply:
We should have some new merges soon!

Reply:
Just saw some new commits, this PR would be ready in a couple of hours.